receiver/postgresqlreceiver/metadata.yaml

```
@@ -0,0 +1,69 @@
+name: postgresql
+
+labels:
+  database:
+    description: The name of the database.
+  table:
+    description: The schema name followed by the table name.
+  source:
+    description: The block read source type.
+    enum: [ heap_read, heap_hit, idx_read, idx_hit, toast_read, toast_hit, tidx_read, tidx_hit ]  
+  operation:
+    description: The database operation.
+    enum: [ seq, seq_tup_read, idx, idx_tup_fetch ]
+  state:
+    description: The tuple (row) state either dead or live
+    enum: [ dead, live ]
+
+
+metrics:
+  postgresql.blocks_read:
+    description: The number of blocks read.
+    units: 1
+    data:
+      type: int sum
+      monotonic: true
+      aggregation: cumulative
+    labels: [ database, table, source ]
+  postgresql.commits:
+    description: The number of commits.
+    units: 1
+    data:
+      type: int sum
+      monotonic: true
+      aggregation: cumulative
+    labels: [ database ]
+  postgresql.db_size:
+    description: The database disk usage.
+    units: By
+    data:
+      type: gauge
+    labels: [ database ]
+  postgresql.backends:
+    description: The number of backends.
+    units: 1
+    data:
+      type: gauge
+    labels: [ database ]
+  postgresql.db_rows:
```

```suggestion
  postgresql.rows:
```
The `db_` prefix isn't necessary. In fact, it may be a bit misleading, since each datapoint actually describes a table.

---

fixed